web/src/apps/main/redux/reducers/index.ts

```diff
@@ -10,6 +10,7 @@ import { learnPage } from "./learn-page";
 import { navbarComponent } from "./navbar-component";
 import { projectsPage } from "./projects-page";
 import { settings } from "./settings";
+import { team } from "./team";
```

Comment:
let's be consistent:

```suggestion
import { teamPage } from "./team-page";
```